api/clients/v2/accountant.go

```diff
@@ -17,18 +16,21 @@ type Accountant struct {
 	// on-chain states
 	accountID    gethcommon.Address
 	reservations map[core.QuorumID]*core.ReservedPayment
-	onDemand     *core.OnDemandPayment
+	// OnDemand initially only enabled on quorum. Accountant must be updated to be quorum specific
```

Comment:
"on quorum" -> missing quorums 0 and 1?

Reply:
actually read somewhere else that its only on quorum 0. I'm curious as to why. Is that documented in our docs site?

Reply:
the existing payments only have one on-demand balance per account, and that balance is used for either quorum 0 or 1 offchain. 
in the new version of payments i.e. UsageAuthorizationRegistry, we are going use a new interface that allows contract owner to enable quorum specific depositing/balances for custom quorums, but initially we will still have just one balance per account to keep UI and offchain accounting the same, and we picked quorum 0 for this purpose. 
I think the docs should update for the new updates but not now